modules/llm-cache/storage/local_file_storage.cc

```diff
@@ -144,4 +156,24 @@ bool LocalFileStorage::IsFileExist(const std::string& path) {
   return std::filesystem::exists(path);
 }
 
+Status LocalFileStorage::Delete(std::string path) {
+  if (std::filesystem::exists(path)) {
+    std::filesystem::remove_all(path);
+  }
+  return Status::OK();
+}
+
+std::string LocalFileStorage::GetTmpFileDir(std::string filePath) {
+  pid_t pid = getpid();
+  return this->tempFileDir + std::to_string(pid);
+}
+
+Status LocalFileStorage::MoveFileAtomic(std::string src, std::string dst) {
+  if (renameat2(AT_FDCWD, src.c_str(), AT_FDCWD, dst.c_str(),
```

Comment:
Could you please make it compatible with a lower version of Linux kernel like CentOS7. While building it from centos7 image I get the following error.

```bash
#0 21.17 /work/v6d/modules/llm-cache/storage/local_file_storage.cc:174:17: error: ‘RENAME_NOREPLACE’ was not declared in this scope
#0 21.17   174 |                 RENAME_NOREPLACE)) {
#0 21.17       |                 ^~~~~~~~~~~~~~~~
#0 21.17 /work/v6d/modules/llm-cache/storage/local_file_storage.cc:173:7: error: ‘renameat2’ was not declared in this scope; did you mean ‘renameat’?
#0 21.17   173 |   if (renameat2(AT_FDCWD, src.c_str(), AT_FDCWD, dst.c_str(),
#0 21.17       |       ^~~~~~~~~
#0 21.17       |       renameat
```